engine/src/deploy_sign.cpp

```diff
@@ -1034,7 +1034,8 @@ static bool MCDeploySignWindowsAddTimeStamp(const MCDeploySignParameters& p_para
 	// the sig.
 	if (t_request != nil)
 	{
-		t_request -> blob -> signature = nil;
+        if (t_request -> blob != nil)
```

Comment:
This would probably be easier to read if you combined the `if` conditions, i.e.:

    if (t_request != nil && t_request->blob != nil)
        SpcTimeStampRequest_free(t_request);